sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncUserDefinedFunction.java

```diff
@@ -73,7 +73,7 @@ CosmosAsyncUserDefinedFunction setId(String id) {
     public Mono<CosmosAsyncUserDefinedFunctionResponse> replace(CosmosUserDefinedFunctionProperties udfSettings) {
         return container.getDatabase()
                    .getDocClientWrapper()
-                   .replaceUserDefinedFunction(new UserDefinedFunction(udfSettings.toJson()), null)
+                   .replaceUserDefinedFunction(new UserDefinedFunction(ModelBridgeInternal.toJsonFromJsonSerializable(udfSettings)), null)
```

Comment:
(Could be out of scope of this PR) Potential double serialization. Instead of converting toJson and again back from Json, we could just use the propertybag to get UDF by using Resource(ObjectNode) overload.

Reply:
Will discuss offline on this one with you.

Reply:
outside of scope of this PR. we can come back to this later. I fixed double serialization for point read item operation but not the udf, etc.